Stats: NPE when opening the Stats view

### Expected behavior
Stats should be shown

### Actual behavior
NPE  on [this line ](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsInsightsLatestPostSummaryFragment.java#L64)

```
06-07 14:09:27.316 13054-13054/org.wordpress.android E/AndroidRuntime: FATAL EXCEPTION: main
                                                                       Process: org.wordpress.android, PID: 13054
                                                                       de.greenrobot.event.EventBusException: Invoking subscriber failed
                                                                           at de.greenrobot.event.EventBus.handleSubscriberException(EventBus.java:518)
                                                                           at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:500)
                                                                           at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492)
                                                                           at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67)
                                                                           at android.os.Handler.dispatchMessage(Handler.java:102)
                                                                           at android.os.Looper.loop(Looper.java:135)
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5254)
                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                           at java.lang.reflect.Method.invoke(Method.java:372)
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903)
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698)
                                                                        Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean org.wordpress.android.ui.stats.models.InsightsLatestPostModel.isLatestPostAvailable()' on a null object reference
                                                                           at org.wordpress.android.ui.stats.StatsInsightsLatestPostSummaryFragment.onEventMainThread(StatsInsightsLatestPostSummaryFragment.java:64)
                                                                           at java.lang.reflect.Method.invoke(Native Method)
                                                                           at java.lang.reflect.Method.invoke(Method.java:372)
                                                                           at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:498)
                                                                           at de.greenrobot.event.EventBus.invokeSubscriber(EventBus.java:492) 
                                                                           at de.greenrobot.event.HandlerPoster.handleMessage(HandlerPoster.java:67) 
                                                                           at android.os.Handler.dispatchMessage(Handler.java:102) 
                                                                           at android.os.Looper.loop(Looper.java:135) 
                                                                           at android.app.ActivityThread.main(ActivityThread.java:5254) 
                                                                           at java.lang.reflect.Method.invoke(Native Method) 
                                                                           at java.lang.reflect.Method.invoke(Method.java:372) 
                                                                           at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:903) 
                                                                           at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:698) 
06-07 14:09:27.359 13054-13193/org.wordpress.android V/WordPress-STATS: Parsing the following Timezone received from WP: 0
```

### Steps to reproduce the behavior
Open the app and go to Stats. Happens consistently with any site.

##### Tested on Genmotion Nexus 6, Android 5.1.0

--- a/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsInsightsLatestPostSummaryFragment.java
+++ b/WordPress/src/main/java/org/wordpress/android/ui/stats/StatsInsightsLatestPostSummaryFragment.java
@@ -56,6 +56,11 @@ public class StatsInsightsLatestPostSummaryFragment extends StatsAbstractInsight
             return;
         }
 
+        if (event.mInsightsLatestPostModel == null) {
+            showErrorUI();
+            return;
+        }
+
         mInsightsLatestPostModel = event.mInsightsLatestPostModel;
 
         // check if there is a post "published" on the blog
@@ -83,6 +88,11 @@ public class StatsInsightsLatestPostSummaryFragment extends StatsAbstractInsight
     @SuppressWarnings("unused")
     public void onEventMainThread(StatsEvents.InsightsLatestPostDetailsUpdated event) {
         if (!shouldUpdateFragmentOnUpdateEvent(event)) {
+            return;
+        }
+
+        if (mInsightsLatestPostModel == null || event.mInsightsLatestPostDetailsModel == null) {
+            showErrorUI();
             return;
         }
 

